app/code/Magento/InventoryInStorePickupApi/Api/NotifyOrderIsReadyAndShipInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickupApi\Api;
+
+interface NotifyOrderIsReadyAndShipInterface
```

Comment:
why the method for In-Store Pickup API contains "And Ship" part? 
While we don't have "real" shipment there

Reply:
You are right. Fixed.